### PR TITLE
GraphQL enhancements

### PR DESCRIPTION
**Pull request by avasquez614**
* Provided contentItems, pages and components fields, which can be use to query all content items, all pages and all components respectively.

Ticket craftercms/craftercms#3078

### CODE REVIEW

File: src/main/java/org/craftercms/engine/graphql/SchemaUtils.java [hunk 1]
```diff
@@ -17,15 +17,10 @@
...
-import java.util.List;
+import java.util.*;
 
-import graphql.schema.GraphQLArgument;
-import graphql.schema.GraphQLEnumType;
-import graphql.schema.GraphQLFieldDefinition;
-import graphql.schema.GraphQLInputObjectField;
-import graphql.schema.GraphQLInputObjectType;
-import graphql.schema.GraphQLObjectType;
+import graphql.schema.*;
```
Comment (sumerjabri) [inline src/main/java/org/craftercms/engine/graphql/SchemaUtils.java]: Please don't include with *

Comment (avasquez614) [reply src/main/java/org/craftercms/engine/graphql/SchemaUtils.java]: That's IntelliJ's optimizer, when there are too many imports in that package it includes all, I would have to manually enter each one then.

**Comment by sumerjabri** [comment]
@avasquez614 please review Codacy and respond here.

**Comment by avasquez614** [comment]
Codacy fixes pushed

**Pull request merged**